00-Raster-Structure.Rmd: ZB review edits

brymz opened the pull request:
Most of the changes are simple text edits or word smithing. I updated the UTM picture from wikipedia that got deleted/moved. I also moved some text out of figure captions because I felt it was oddly formatted or lost emphasis as a caption.

Great work putting this together, @lwasser & @mjones01!

lwasser commented on 00-Raster-Structure.Rmd:
```diff
@@ -83,9 +81,7 @@ NEON Data Skills: Image Raster Data in R - An Intro</a>
-Raster or "gridded" data are saved on a regular grid which is rendered on a map
-as pixels. Each pixel contains a value that represents an area on the Earth's 
-surface.
+Raster or "gridded" data are stored as a grid of values which are rendered on a map as pixels. Each pixel value represents an area on the Earth's surface.
```
hi @brymz . I should document this online however we stick to a 80 character width limit on our md / Rmd files like data and software carpentry do. If you don't have time to adjust this, i can easily do it after I merge. It looks like this is all on one line.

lwasser commented on 00-Raster-Structure.Rmd:
```diff
@@ -460,20 +449,20 @@ is likely caused by an error in either data collection or processing.
 
 We can explore the distribution of values contained within our raster using the 
-`hist` function which produces a histogram. Histograms are often useful in 
+`hist()` function which produces a histogram. Histograms are often useful in 
```
@brymz I like this approach of using `hist()` (vs `hist`). @mjones01 and i went back and forth on it as i was using the latter and she was using the former. Do you think we should do this throughout for all functions? If so, i'll post an issue to address in our other lesson repos too!

lwasser commented on 00-Raster-Structure.Rmd:
```diff
@@ -64,13 +64,11 @@ preferably RStudio, loaded on your computer.
 {% include/dataSubsets/_data_Airborne-Remote-Sensing.html %}
 
-****
+## Things To Know To Complete This Lesson
```
@mjones01 will you have a look at this before I accept? Is this format in line with our other grey boxes? @brymz just FYI we spent some time making the grey boxes at the top consistent so i'm hesitant to make this change as small as it is because then we have to change ALL of our other lessons. In the future my plan is to automate this box entirely :)

mjones01 replied:
It is not inline with the current model but I think it is a good suggestion to think about.  I'm going to add it to the Issue (https://github.com/NEONInc/NEON-Data-Skills-Development/issues/3) in the Data-Skills-Dev repo that is also about grey box styling.  For now I'd not include this suggestion but we will revisit it when we can implement it across the site.

lwasser commented:
thank you @brymz ! These edits look great! I have a few MINOR comments:

1. Line Width: we use 80 characters throughout for all lessons. How do you recommend we communicate that? maybe in a "contributing" document on the repo? 
2. grey box - i looped in Megan as we did some work to standardize that across all of our lessons and i'm hesitant to change it here and then have to change it everywhere :) if the rules look better, then we can make an issue and do this across all lessons in the near future to make sure it's all consistent. 

THANK YOU so much for your time in editing these -

brymz commented:
> How do you recommend we communicate 80 characters?

Oops. 80 characters, absolutely. I keep my text editor at 80 characters so I can easily make the hard wrap, but sometimes I miss that step when I'm editing content. Thanks for the gentle reminder.
As for the bigger question, I developed a style guide for the Data Carpentry semester course (http://www.datacarpentry.org/semester-biology/docs/style-guide/) that I shamelessly used here. The `function()` formatting is also addressed there. I would definitely recommend a `contributing.md` and/or README.md that outlines these sort of topics. 

> grey box 

I totally get avoiding making sweeping changes. It struck me that the section broken up by `****` was out of place or lacked clarity. No strong feelings about it though. I went back and changed throughout.

mjones01 commented:
Added @brymz  comment about the header within the grey box to an existing Issue about formatting the grey box.  https://github.com/NEONInc/NEON-Data-Skills-Development/issues/3

lwasser commented:
thank you @brymz ! merging this.
